swapi/src/app/app.module.ts

```diff
@@ -0,0 +1,41 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+
+import { AppRoutingModule } from './app-routing.module';
+import { AppComponent } from './app.component';
+import { SignInComponent } from './sign-in/sign-in.component';
+import { SignUpComponent } from './sign-up/sign-up.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { FormsModule, ReactiveFormsModule } from "@angular/forms";
+import { MatFormFieldModule } from "@angular/material/form-field";
+import { MatIconModule } from "@angular/material/icon";
+import { MatDialogModule } from "@angular/material/dialog";
+import { MatButtonModule } from "@angular/material/button";
+import { MatInputModule } from "@angular/material/input";
+
+const materialModules = [
+  MatFormFieldModule,
+  MatIconModule,
+  MatDialogModule,
```

Comment:
looks unused

Reply:
these modules are used in the both forms. material directives won't work if delete

Reply:
I left comment under matDialog module, are useing matDialog somewhere ?

Reply:
deleted